src/components/people/PersonCard/index.tsx

```diff
@@ -70,6 +71,21 @@ export default ({
             {currentPerson && (
                 <div className={containerClassNames}>
                     <PersonPhoto person={currentPerson} size={photoSize} hidePopover />
+                    {isFetching && (
```

Comment:
If I understand this code correctly, the skeleton will never show since `isFetching `will always be false when `currentPerson `is set?